.gitignore

```diff
@@ -144,6 +144,7 @@ DocProject/Help/*.hhk
 DocProject/Help/*.hhp
 DocProject/Help/Html2
 DocProject/Help/html
+**/doc/generated_doxygen_docs/**
```

Comment:
Move this down to the doxygen section and replace `/[Dd]ocs/`:
https://github.com/Azure/azure-sdk-for-c/blob/283f8c80518db48c604daf74bdbe81f273795e1b/.gitignore#L305-L307

Reply:
Why the `**`?
```suggestion
/doc/generated_doxygen_docs/
```